### DISCUSSION TITLE
Visual Studio Copilot Chat Extension, Code Output too narrow to read

### DISCUSSION THREAD

**Discussion by YouKnowThem** [community/community#83930] [Copilot Conversations]
### Select Topic Area

Bug

### Body

See:
https://stackoverflow.com/questions/77745101/visual-studio-copilot-chat-extension-code-output-too-narrow-to-read

**Comment by ayim** [comment]
@rheapatel23's the expert on Chat in Visual Studio, what do ya think?

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com

**Comment by rheapatel23** [comment]
Hi @YouKnowThem , please report a bug here: https://learn.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2022


The GitHub Copilot Visual Studio team will take a look :) 

Thanks, 
Rhea - Product Manager @ Microsoft

**Comment by YouKnowThem** [comment]
Thank you for pointing me in the right direction. After I found this:
https://developercommunity.visualstudio.com/t/The-Copilot-Chat-extensions-code-boxes-/10364048#T-N10559517
I updated my GitHub Copilot Chat Extension version and the code output was readable (again).

Only downside that my previous Copilot Chat Conversations were gone and could not be found again.

@rheapatel23 please update your answer on stackoverflow to include this link, then I will accept your answer there. Thank you very much and have a nice day.
